src/NuGet.Clients/NuGet.PackageManagement.UI/Options/GeneralOptionControl.cs

```diff
@@ -62,11 +62,16 @@ internal void OnActivated()
                 catch (InvalidOperationException)
                 {
                     MessageHelper.ShowErrorMessage(Resources.ShowError_ConfigInvalidOperation, Resources.ErrorDialogBoxTitle);
-               } 
+                } 
                 catch (UnauthorizedAccessException)
                 {
                     MessageHelper.ShowErrorMessage(Resources.ShowError_ConfigUnauthorizedAccess, Resources.ErrorDialogBoxTitle);
                 }
+                catch (Exception ex)
```

Comment:
i do not agree that we need to catch ALL the exceptions. this is just going to hide any possible exceptions that get thrown from the above code path.

have you been able to determine what exception is being thrown and why?

Reply:
any exception from option UI crash VS, we did this three time, every time we caught a specified exception, we got a watson crash later. we should make sure nuget do not crash VS first and it wnot hides the exception, VS pops up a error window and log the exception to activity log

Reply:
are we able to look at the exceptions in activity log the same way we can look at Watson results?

Reply:
Yes, we will have full stack in activity log, we already did this for applying setting, just missed the init part before.

Reply:
no what I mean is  that is the activity log available to us without people filing a feedback ticket?